DEEP_DIVES/46-building-dynamic-p2p-applications.md

```diff
@@ -1,15 +1,30 @@
 # Building Dynamic P2P Applications
 
-<Short summary of topic including Open Problem(s) to solve>
+IPFS and its associated technologies are mostly *static*. Radicle is an attempt to build 
+a *dynamic* P2P web on top of IPFS; one which can host applications with the same abilities
+of the centralized servers of today to receive inputs from various people or sources. 
+
 
 ## Goal
 
-<Define what we would get if the Open Problem(s) are solved for this Deep Dive>
+We've encountered a few difficulties in making this work smoothly. IPFS pubsub in particular
+is quite slow. One big advantage for us would be to be able to control the network topology
```

Comment:
Would you be able to elaborate on "quite slow" - what aspect in particular? e.g. signing and sending a message? receiving a message from a node that you're not directly connected to?